NuRadioReco/test/test_examples.sh

```diff
@@ -1,12 +1,12 @@
 set -e
 cd NuRadioReco/examples/PhasedArray/Effective_volume
-python T01generate_event_list.py minimal
```

Comment:
Yes, it changes nothing for the build bot. It is just more convenient if you want to run the tests on your own system. With this change it will run the correct python version even if py3 is not the default.